src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java

```diff
@@ -1,50 +1,65 @@
 package com.wikia.webdriver.common.core.configuration;
 
-import com.wikia.webdriver.common.core.exceptions.TestEnvInitFailedException;
-import com.wikia.webdriver.common.properties.Credentials;
-
-import org.apache.commons.lang.StringUtils;
-import org.yaml.snakeyaml.Yaml;
-
 import java.io.File;
 import java.io.FileInputStream;
 import java.io.FileNotFoundException;
-import java.io.InputStream;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.logging.Logger;
+
+import org.apache.commons.lang.StringUtils;
+import org.yaml.snakeyaml.Yaml;
+
+import com.wikia.webdriver.common.core.exceptions.TestEnvInitFailedException;
+import com.wikia.webdriver.common.properties.Credentials;
 
 /**
  * Configuration handler. This Class should handle run configuration and global properties.
- * Configuration handling: 1. If there is a System Property specified - return value of this
- * property 2. If no System Property is found - value is provided from configuration files
- * (config_sample.yml if no config.yml provided) NOTE: system property should override ANY property
- * specified in config files
- */
+ * Configuration handling:
+ * <ol>
+ * <li>Look for the property key in testConfig map, if key is present, return the value</li>
+ * <li>Look for the property key in system properties - return value of this property if key present
+ * </li>
+ * <li>If no System Property is found - value is provided from configuration files
+ * (config_default.yml and config.yml). Values provided in config.yml, are overriding values from
+ * config_default.yml</li>
+ * </ol>
+ * */
 public class Configuration {
 
-  private static Map<String, String> config;
+  private static final String DEFAULT_CONFIG_FILE_NAME = "config_default.yml";
+  private static final String LOCAL_CONFIG_FILE_NAME = "config.yml";
+  private static final Logger LOGGER = Logger.getLogger(Configuration.class.getName());
+
+  private static Map<String, String> defaultConfig;
+  private static Map<String, String> localConfig = new HashMap<>();
   private static Map<String, String> testConfig = new HashMap<>();
 
-  private Configuration() {
-  }
+  private Configuration() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)